src/extensions.js

```diff
@@ -5,6 +5,7 @@ import './utils/block-category';
 import './js/coblocks-color-fix';
 
 // Extensions
+import './extensions/apply-extensions';
```

Comment:
The other extensions import styles directly here in some cases. To prevent the import now would beak existing styles.

Reply:
I've identified in another commit where this is an issue while fixing its side-effects.

https://github.com/godaddy-wordpress/coblocks/pull/2196/files#diff-715fee81e18461db0b1a0f37d048f8277f43cb3b568654b48c32d31444ac8687R1-R7

Reply:
Since I have taken time to remove the excess style imports I feel comfortable removing the superfluous imports now.